docs/api/v1.md

```diff
@@ -380,49 +382,82 @@ Responses:
 }
 ```
 
-### Function
+### SourceFunction
 ```
 {
    address {integer}: Memory address of the function
    name {string}: Name of the function
    sourcePath {string}: Full path to the source code
    lineNumber {integer}: Line number of the function definition
    lineCount {integer}: Length of function in source code lines
-   parameters {Variable[]}: Array of parameter variables
-   localVariables {Variable[]): Array of all local variables defined in function
+   parameters {SourceVariable[]}: Array of parameter variables
+   localVariables {SourceVariable[]): Array of all local variables defined in function
 }
 ```
 
 ### DebugInfo
 ```
 {
   id {DebugId}
-  attachedProcess {[Process](#process)}
+  attachedProcess {Process}
   sourcePath {string}: Path to root directory of source code
 }
 ```
 
 ### Breakpoint
 ```
 {
-  function {[Function](#Function)}: The function that this is breakpoint is associated with.
+  sFunction {SourceFunction}: The function that this is breakpoint is associated with.
   metadata {string}: Any additional info associated with this breakpoint.
 }
 ```
 
-### Variable
+### SourceVariable
 ```
 {
   id {var id}: Unique identifier of that variable.
   name {string}: Name of the variable as it appears in the function.
-  sType {SourceType}: Type of the variable as it is defined in the source code.
+  sValueIndex {number}: Index to the variable in the SourceValue[] as it is defined in the source code.
```

Comment:
Can't define a single value index for a `SourceVariable`, since they change over the course of a trace, and `SourceVariable`s are part of a function definition. Trace objects refer back to `SourceVariable`s by index instead.